src/test/java/jp/co/yahoo/yosegi/spread/column/TestPrimitiveCellManager.java

```diff
@@ -0,0 +1,346 @@
+
+  @Test
+  public void T_size_1() throws IOException {
+    ICellMaker maker = CellMakerFactory.getCellMaker( ColumnType.STRING );
+    PrimitiveCellManager cellManager = new PrimitiveCellManager( maker );
+    cellManager.add( new StringObj( "test" ) , 0 );
+
+    assertEquals( cellManager.size() , 1 );
+  }
+
+  @Test
+  public void T_size_2() throws IOException {
+    ICellMaker maker = CellMakerFactory.getCellMaker( ColumnType.STRING );
+    PrimitiveCellManager cellManager = new PrimitiveCellManager( maker );
+    cellManager.add( new StringObj( "test" ) , 5 );
+
+    assertEquals( cellManager.size() , 6 );
+  }
+
+  @Test
+  public void T_clear_1() throws IOException {
+    ICellMaker maker = CellMakerFactory.getCellMaker( ColumnType.STRING );
+    PrimitiveCellManager cellManager = new PrimitiveCellManager( maker );
+    cellManager.add( new StringObj( "test" ) , 5 );
+    assertEquals( cellManager.size() , 6 );
+
+    cellManager.clear();
+    assertEquals( cellManager.size() , 0 );
+  }
+
+  @Test
+  public void T_filter_1() throws IOException {
+    ICellMaker maker = CellMakerFactory.getCellMaker( ColumnType.STRING );
+    PrimitiveCellManager cellManager = new PrimitiveCellManager( maker );
+    cellManager.add( new StringObj( "test" ) , 5 );
+
+    boolean[] filters = new boolean[cellManager.size()];
+    assertEquals( filters.length , 6 );
+
+    assertEquals( filters[0] , false );
+    assertEquals( filters[1] , false );
+    assertEquals( filters[2] , false );
+    assertEquals( filters[3] , false );
+    assertEquals( filters[4] , false );
+    assertEquals( filters[5] , false );
+    cellManager.filter( new NotNullFilter( ColumnType.STRING ) , filters );
+    assertEquals( filters[0] , false );
+    assertEquals( filters[1] , false );
+    assertEquals( filters[2] , false );
+    assertEquals( filters[3] , false );
+    assertEquals( filters[4] , false );
+    assertEquals( filters[5] , true );
+  }
+
+  @Test
+  public void T_filter_2() throws IOException {
+    ICellMaker maker = CellMakerFactory.getCellMaker( ColumnType.STRING );
+    PrimitiveCellManager cellManager = new PrimitiveCellManager( maker );
+    cellManager.add( new StringObj( "test" ) , 1 );
+    cellManager.add( new StringObj( "test" ) , 3 );
+    cellManager.add( new StringObj( "test" ) , 5 );
+
+    boolean[] filters = new boolean[cellManager.size()];
+    assertEquals( filters.length , 6 );
+
+    assertEquals( filters[0] , false );
+    assertEquals( filters[1] , false );
+    assertEquals( filters[2] , false );
+    assertEquals( filters[3] , false );
+    assertEquals( filters[4] , false );
+    assertEquals( filters[5] , false );
+    cellManager.filter( new NotNullFilter( ColumnType.STRING ) , filters );
+    assertEquals( filters[0] , false );
+    assertEquals( filters[1] , true );
+    assertEquals( filters[2] , false );
+    assertEquals( filters[3] , true );
+    assertEquals( filters[4] , false );
+    assertEquals( filters[5] , true );
+  }
+
```

Comment:
Please describe what are tested in the test method separately at T_filter_?.
For example, T_filter_when_xxx or some.

Reply:
Please add a prefix or suffix like T_ or Text or some so that the code reader knows it is a test method including assertThat or assertThrows and/or some testing methods.